docs/banzai_nres/index.rst

```diff
@@ -8,3 +8,135 @@ Reference/API
 =============
 
 .. automodapi:: banzai_nres
+
+
+
+Running Code Tests Locally
+==========================
+
+To test code style: "tox -e codestyle"
+To test documentation: "tox -e build_docs"
+To test the functionality of the code itself (on python 3.8): "tox -e py38-test-alldeps"
+
+Output Data Products
+====================
+
+Here we cover the various quantities contained in the banzai-nres output data products.
+
+
+Master wavelength calibration files
+-----------------------------------
+
+Master wavelength calibrations are stacked arc (double) exposures, wavelength calibrated primarily
+with xwavecal routines. These file names follow the convention `site-camera-date-double-binmode-litfibers.fits.fz`
+E.g. lscnrs01-fa09-20190521-lampflat-bin1x1-110.fits.fz for an observation taken at lsc using the Archon camera on
+2019/05/21 in 1x1 binning mode with fibers 0 and 1 lit.
+
+
+Wavelength calibration metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In each master wavelength calibration file, the primary hdu header holds four wavelength metrics. The keys for
+these are:
+
+* 'SIGLAM'
+* 'PRECISN'
+* 'CHISQ'
+* 'LINENUM'
+* 'MLINENUM'
+
+Before we launch into detail in each, we have to define what a 'matched line' is. A matched line is a feature on the
```

Comment:
I feel like we could streamline this to make it more readable. "Matched lines are any detected features that have a corresponding line in the atlas within 0.1 Anstgroms."